modules/devguide/examples/c/subdoc-updating.cc

```diff
@@ -106,22 +108,23 @@ main()
         lcb_SUBDOCSPECS* specs = nullptr;
         check(lcb_subdocspecs_create(&specs, 3), "create SUBDOC operations container");
 
+        // tag::path[]
         std::vector<std::string> paths{
             "array",
             "array[0]",
             "description",
         };
-
+        // tag::array-insert[]
```

Comment:
Just a quick sanity check: Does Antora do the right thing with these nested tags here?

Reply:
I don't know. It was in the original PR. If not, then we should fix the sample (extract the piece or something like that). @RichardSmedley, do you know?

Reply:
The array-insert tag is not used in the adoc file, so could safely be deleted.